src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -170,11 +172,21 @@ export const CheckoutPayment = ({
     { loading: isBundleCreationInProgress },
   ] = useCreateQuoteBundleMutation()
   const [startCheckout] = useStartCheckoutMutation()
+  const [addPaymentTokenMutation] = useAddPaymentTokenMutation()
   const [getStatus] = useCheckoutStatusLazyQuery({
     pollInterval: 1000,
   })
 
+  const { payment: is3DsComplete } = useParams<{ payment: string }>()
   const onConnectPaymentSuccess = useCallback(async () => {
+    const paymentTokenId = storage.session.getSession()?.paymentTokenId
+    if (!paymentTokenId) throw new Error('No token payment id')
```

Comment:
Why not pass this as a parameter to the function instead?

Reply:
added this as a parameter to onConnectPaymentSuccess.